lib/gettext-setup/gettext_setup.rb

```diff
@@ -9,7 +9,9 @@ module GettextSetup
   # - config.yaml
   # - a .pot file for the project
   # - i18n directories for languages, each with a .po file
-  def self.initialize(locales_path)
+  # - if using .mo files, an LC_MESSAGES dir in each language dir, with the .mo file in it
+  # `file_format` - one of the supported backends for fast_gettext (e.g. :po, :mo, :yaml, etc.)
+  def self.initialize(locales_path, file_format = :po)
```

Comment:
Can we use a hash argument pattern here? Second argument could be e.g. `options = {}` and use `options[:file_format] || :po` below. That allows invocation like `initialize(locales_path, file_format => :yaml)` and extensibility in arguments later, without needing to pass `nil` to skip an argument.